src/Context/Traits/UserAwareContextTrait.php

```diff
@@ -84,31 +85,134 @@ public function loggedIn()
         return false;
     }
 
-
-
     /**
      * Create a user.
+     * If the user already exists, the existing user will be
```

Comment:
Needs to be a line between the PHPDoc summary and PHPDoc description lines. See https://docs.phpdoc.org/references/phpdoc/basic-syntax.html

Reply:
Hmmm.. I wonder if there's a way of getting phpcs to check phpdoc syntax too. I'll add it to the list :)